lib/components/post-summary/post-summary.a11y.test.ts

```diff
@@ -0,0 +1,183 @@
+import { html } from "@open-wc/testing";
+import { runA11yTests } from "../../test/a11y-test-utils";
+import { getChildren } from "../../test/post-summary-test-utils";
+import type {
+    BadgeType,
+    TruncationSizes,
+} from "../../test/post-summary-test-utils";
+import "../../index";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const template = ({ component, testid }: any) => html`
+    <div class="d-flex ai-center jc-center p8 ws2" data-testid="${testid}">
+        ${component}
+    </div>
+`;
+
+describe("post-summary", () => {
+    // Base, sparce
+    runA11yTests({
+        baseClass: "s-post-summary",
+        modifiers: {
+            primary: ["deleted", "ignored", "watched"], // variants described as modifiers to test colliding modifiers
+            secondary: ["minimal"],
+        },
+        children: {
+            default: getChildren({
+                show: {
+                    description: true,
+                    menuBtn: true,
+                    stats: true,
+                    tags: true,
+                    title: true,
+                    user: true,
+                },
+            }),
+            sparce: getChildren({
+                show: {
+                    stats: true,
+                    tags: true,
+                    title: true,
+                    user: true,
+                },
+                tags: [
+                    {
+                        text: "featured-request",
+                        type: "required",
+                    },
+                ],
+                title: "Short title",
+            }),
+        },
+        template,
+        // TODO resolve test failures
+        skippedTestids: [
```

Comment:
Do you think these failures deserve to be logged in our a11y board?

Reply:
Yep, I'll add them

Reply:
- Deleted [A11Y-27](https://stackoverflow.atlassian.net/browse/A11Y-27), [A11Y-28](https://stackoverflow.atlassian.net/browse/A11Y-28)
- Ignored [A11Y-34](https://stackoverflow.atlassian.net/browse/A11Y-34), [A11Y-35](https://stackoverflow.atlassian.net/browse/A11Y-35)
- Watched [A11Y-36](https://stackoverflow.atlassian.net/browse/A11Y-36)